astropy/stats/src/fast_sigma_clip.c

```diff
@@ -27,7 +27,7 @@ static PyMethodDef module_methods[] = {{NULL, NULL, 0, NULL}};
   ob = PyModule_Create(&moduledef);
 
 MOD_INIT(_fast_sigma_clip) {
-    PyObject *m, *d;
+    PyObject *m, *d = NULL;
```

Comment:
My C skills might be rusty as I fail to see why this assignment is needed (or maybe I am not expanding all the macros in my head). Why are `MOD_INIT` and `MOD_DEF` macros are necessary at all? Why not expand them?

Reply:
I see, it is because of `goto fail` in the `if` before `d` gets initialized.